src/GraphQLMiddlewareDelegatorFactory.php

```diff
@@ -8,11 +8,11 @@
 
 namespace GraphQLMiddleware;
 
-
 use GraphQLMiddleware\Error\JsonErrorResponseGenerator;
 use Interop\Container\ContainerInterface;
 use Zend\Diactoros\Response;
-use Zend\ServiceManager\Factory\DelegatorFactoryInterface;
+use Zend\ServiceManager\DelegatorFactoryInterface;
```

Comment:
this is deprecated. thats why i implemented `Zend\ServiceManager\Factory\DelegatorFactoryInterface`. If you take a look at the code in: https://github.com/zendframework/zend-servicemanager/blob/master/src/DelegatorFactoryInterface.php#L13